### ISSUE TITLE
Can knitrBootstrap make slide decks (ioslides/slidy)?

### ISSUE THREAD

**Issue by nathancarter** [jimhester/knitrBootstrap#190]
This project looks great, thank you for making it.  My only question is whether also works for slide decks (ioslides or slidy, the two that come built into RStudio).
 * Does this project already support that feature and I just missed it?
 * If not, is such support planned?
 * If not, do you have tips on what I should do to extend this project to support ioslides/slidy?
 * Or, if you're aware of another project that meets these needs, that would be great, too.

Thank you very much.

**Comment by jimhester** [comment]
What knitrBootstrap feature are you looking for in a slide deck? Rmarkdown supports nearly all of the features of knitrBootstrap now including [floating toc](http://rmarkdown.rstudio.com/html_document_format.html#floating_toc) and [code folding](http://rmarkdown.rstudio.com/html_document_format.html#code_folding).

Development on knitrBootstrap is mostly stalled at this point, I have no plans to work on slide support now or in the future.

If you tell me what features you are looking for I can try to point you in the right direction though.

**Comment by nathancarter** [comment]
The styles!  ioslides and slidy seem to have one style each, and I don't like either.  Details at [this StackOverflow question](http://stackoverflow.com/questions/36605634/in-rstudio-making-slide-deck-with-rmarkdown-how-can-i-easily-change-the-theme?noredirect=1#comment60808819_36605634).

**Comment by nathancarter** [comment]
Whoops...wrong button.  Of course, you can close this issue if you want.

**Comment by jimhester** [comment]
You can supply a custom css to both ioslides and slidy and customize the theme how you like.

If you are looking for pre-existing themes both [revealjs](http://rmarkdown.rstudio.com/revealjs_presentation_format.html) and [beamer](http://rmarkdown.rstudio.com/beamer_presentation_format.html) have a number of themes to pick from.

**Comment by nathancarter** [comment]
Reveal isn't respecting slide_level for me, but I can work on that and check their docs/support.

But supplying a custom css to ioslides/slidy is only good if I have a custom css to provide.  I took the bootstrap css files linked to in [your source code](https://github.com/jimhester/knitrBootstrap/blob/master/R/knit_bootstrap.R#L25), which depend on fonts specified by relative paths, so I manually modified them to absolute URLs.  Now I can get slidy to compile using those files.  That may not have been what you were intending :) but I think I'm all set now.  Thanks!
